src/ContentMessages.ts

```diff
@@ -580,13 +580,13 @@ export default class ContentMessages {
         } catch (error) {
             // 413: File was too big or upset the server in some way:
             // clear the media size limit so we fetch it again next time we try to upload
-            if (error?.httpStatus === 413) {
+            if (error instanceof HTTPError && error.httpStatus === 413) {
```

Comment:
I agree for interfaces, but i prefer the built-in `instanceof` personally where possible, a typeguard leaves ambiguity on the table in the case of inheritence. If I pass a MatrixError to `isHTTPError` will it return true or false? I'd need to consult the docs, whereas I know `instanceof` will because `MatrixError extends HTTPError`

Reply:
Ah that's an interesting point. I think I generally use the typeguards for simpler cases and hadn't considered inheritance.